### PR TITLE
add configuration docs for 'showHamburgerMenu' and 'showWindowControls'

### PR DESCRIPTION
**Pull request by nickpisacane**
Added documentation for "showHamburgerMenu" and "showWindowControls" to the configuration table. The documentation for the "Property", "Default", and "Description" table values has been derived from the the hyper repository (lib/reducers/ui.js); specifically, lines: [25](https://github.com/zeit/hyper/blob/master/lib/reducers/ui.js#L25), [26](https://github.com/zeit/hyper/blob/master/lib/reducers/ui.js#L26), [85](https://github.com/zeit/hyper/blob/master/lib/reducers/ui.js#L85), [86](https://github.com/zeit/hyper/blob/master/lib/reducers/ui.js#L86). Would be happy to change if I am missing something. Thanks!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -671,8 +671,8 @@ <h2 id="extensions"><a href="#extensions">Extensions</a></h2>
 
         <p><code>Hyper</code> will show a notification when your modules are
         installed to <code>~/.hyper_plugins</code>.</p>
-        
-        <p>You can also take a look at 
+
+        <p>You can also take a look at
```
Comment (matheuss) [inline index.html]: Could you just revert these changes? 😅

**Comment by matheuss** [comment]
Thank you **so much** ☺️ I was supposed to to this before v1.0.0 😅

File: index.html (hunk 1 above)
Comment (nickpisacane) [inline index.html]: No problem! Not sure what happened there.

**Comment by nickpisacane** [comment]
Glad to help, loving the awesome work on Hyper!

**Comment by nickpisacane** [comment]
Updated the branch. Anything else I need to do?

**Comment by matheuss** [comment]
Thank you @nickpisacane 🙌
